src/style.css

```diff
@@ -1 +1,36 @@
-/* styles go here */
+/* head {
+  display: none;
+} */
+
+/* body {
```

Comment:
Unnecessary commented code. Remove the commented body section as it's not needed.